aspnetcore/security/authentication/community.md

```diff
@@ -0,0 +1,16 @@
+# Community OSS Authentication options
+
+When developing ASP.NET Core applications you can use [ASP.NET Core Identity](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/identity) to secure your web applications. You can also replace that with another authentication provider. This page contains community provided open source authentication options for ASP.NET Core. This page will be periodically updated as new providers become available.
```

Comment:
I would remove the first two sentences and just start with "This page contains community . . ."